WombatProject/Startup.cs

```diff
@@ -38,9 +32,11 @@ public void ConfigureServices(IServiceCollection services)
                                       builder.WithOrigins("http://localhost:3000").AllowAnyMethod().AllowAnyHeader();

```

Comment:
This should be come from configuration as well

Reply:
This is going to be fixed a bit later, I will open an issue about it.